relay-kafka/Cargo.toml

```diff
@@ -18,6 +18,8 @@ rmp-serde = { version = "1.1.1", optional = true }
 serde = { version = "1.0.114", features = ["derive"] }
 serde_json = { version = "1.0.55", optional = true }
 thiserror = "1.0.38"
+sentry-kafka-schemas = "0.0.25"
+jsonschema = "0.17.0"
```

Comment:
I wonder if it would make sense to hide the `jsonschema` dependency in `sentry-kafka-schemas`. That is, if we removed this dependency here, and `sentry-kafka-schemas` exposed a `validate(topic, message)` function. 

That way, the library client (Relay) would not have to deal with compiling schemas, and the way that `sentry-kafka-schemas` validates would be completely encapsulated (i.e. you could change the library or even the format it uses for schema definition and validation).

Reply:
See my reply to the local cache comment. For now I would like to avoid that. There's still many unsolved questions around how the workflow in Relay will look like, what performance requirements we have on the entire system, etc.

Reply:
OK! Definitely not a blocker.

> As such it doesn't have many dependencies. This is a good thing if you're only using the generated types in your service.

Makes sense, though this could be solved with cargo features. I.e. there could be a `codegen` feature and a `validation` feature, and clients use either-or. My gut feeling is that 90% of clients that want to perform validation will not care about what schema definition language or validation library is used under the hood, hence the idea to hide it inside the library.

Reply:
i'm reconsidering my response now. one of the reasons i was hesitant to change this was because python currently also works the same (it just exposes the raw schema, doesn't prescribe a schema validation library)

but at the same time it would be good to have a vetted recommended library that everybody ought to use.